### PR TITLE
Created README.md for pangram_checker

### PR DESCRIPTION
**Pull request by SnehaR26**
**Fixes issue:**

**Changes:**

### CODE REVIEW

File: code/string_algorithms/src/pangram_checker/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## PANGRAM
+A pangram is a sentence in which every letter of a given alphabet appears at least once. The most famous english pangram is "The quick brown fox jumps over a lazy dog.".
```
Comment (arnavb) [inline code/string_algorithms/src/pangram_checker/README.md]: No need for two periods in a sentence. Remove the second to last one from the end.

File: code/string_algorithms/src/pangram_checker/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+_We only try to implement pangram checker algorithms for sentences in english language._
```
Comment (arnavb) [inline code/string_algorithms/src/pangram_checker/README.md]: in _the_ english language.

File: code/string_algorithms/src/pangram_checker/README.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+## PANGRAM
```
Comment (arnavb) [inline code/string_algorithms/src/pangram_checker/README.md]: Don't use all caps for titles

File: code/string_algorithms/src/pangram_checker/README.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+A pangram is a sentence in which every letter of a given alphabet appears at least once. The most famous english pangram is "The quick brown fox jumps over a lazy dog.".
+
+_We only try to implement pangram checker algorithms for sentences in english language._
+
+#### PROCEDURE
```
Comment (arnavb) [inline code/string_algorithms/src/pangram_checker/README.md]: Procedure -> Algorithm

Comment (SnehaR26) [reply code/string_algorithms/src/pangram_checker/README.md]: Thank you so much for all the suggestions. I really appreciate it.

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Comment by SnehaR26** [comment]
Thank you so much. I am really liking the journey so far :smile:

**Pull request merged**